docs/src/domains/cdn/_index.md

```diff
@@ -17,17 +17,12 @@ By default, Dedicated projects include a [Fastly CDN managed by Platform.sh](./m
 Self-service Grid plans don't include a CDN by default, but you can set up one at any time,
 such as [Fastly](./fastly.md) or [Cloudflare](./cloudflare.md).
 
-## DNS records
+## Set up your CDN
 
-To start routing client traffic through your CDN,
-some configuration on your DNS are required.
-Such as adding a `TXT` record for the `_acme-challenge` subdomain
-and several `CNAME` records to point to your CDN.
-You can create these records for your domain names through your DNS provider.
-For more information, see you DNS and your CDN provider's official documentations.
+To start routing client traffic through your CDN you need to [set up a custom domain](../steps/_index.md).
 
-Note that [`CNAME` records can't point to apex domains](../steps/dns.md),
-but most CDN providers offer workarounds.
+Note that, if you create `CNAME` records, they [can't point to apex domains](../steps/dns.md).
```

Comment:
💅 So the transition is smoother between the previous sentence and `CNAME` records don't appear suddenly from nowhere without context.

```suggestion
Note that, if you create `CNAME` records when setting up your custom domain, these records [can't point to apex domains](../steps/dns.md).
```